src/app/modules/form/admin-form/admin-form.service.ts

```diff
@@ -670,6 +670,7 @@ export const duplicateForm = (
   newAdminId: string,
   overrideParams: DuplicateFormOverwriteDto,
   workspaceId?: string,
+  duplicateStripped?: boolean,
```

Comment:
If there are several optional params (and we find ourselves having to [include `undefined` as an argument](https://github.com/opengovsg/FormSG/pull/8764/files#diff-d25e48a691b72cce48326009d12cc208e5911e05c08ebcacb1a85651e946eb39R1059)), we could include an options structure like this:

```suggestion
  { workspaceId, duplicateStripped } ?: {
    workspaceId?: string,
    duplicateStripped?: boolean,
  }
```

(we should probably also update the JSDocs for this function with this param)

Reply:
I've updated the function to set this options structure and updated the tests to follow this as well.

Reply:
thanks @scottheng96! 👍